packages/api/cms-api/src/generator/generate-crud-input-json.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { BaseEntity, Entity, MikroORM, PrimaryKey, Property } from "@mikro-orm/core";
+import { LazyMetadataStorage } from "@nestjs/graphql/dist/schema-builder/storages/lazy-metadata.storage";
+import { v4 as uuid } from "uuid";
+
+import { generateCrudInput } from "./generate-crud-input";
+import { lintSource, parseSource } from "./utils/test-helper";
+
+@Entity()
+export class TestEntityWithJson extends BaseEntity<TestEntityWithJson, "id"> {
+    @PrimaryKey({ type: "uuid" })
+    id: string = uuid();
+
+    @Property({ type: "json" })
+    foo: string[] = [];
+
+    @Property({ type: "json" })
+    bar?: number[] = [];
+}
+
+describe("GenerateCrudInputJson", () => {
+    describe("string input class", () => {
+        it("should be a valid generated ts file", async () => {
+            LazyMetadataStorage.load();
+            const orm = await MikroORM.init({
+                type: "sqlite",
+                dbName: "test-db",
+                entities: [TestEntityWithJson],
+            });
+
+            const out = await generateCrudInput({ targetDirectory: __dirname }, orm.em.getMetadata().get("TestEntityWithJson"));
+            //console.log(out);
+            const lintedOutput = await lintSource(out[0].content);
+            console.log(lintedOutput);
```

Comment:
```suggestion
```

Reply:
Imho leaving a commented out console.log in a spot where you usually want to log something when debugging can be helpful.

Reply:
I wasn't aware that you did that on purpose. In the future I will ignore commented out console.log statements.

(We could also think about a logger that supports verbose logging which can be activated via a flag).